environments.go

```diff
@@ -32,10 +32,11 @@ type EnvironmentsService struct {
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/environments.html
 type Environment struct {
-	ID          int    `json:"id"`
-	Name        string `json:"name"`
-	Slug        string `json:"slug"`
-	ExternalURL string `json:"external_url"`
+	ID             int        `json:"id"`
+	Name           string     `json:"name"`
+	Slug           string     `json:"slug"`
+	ExternalURL    string     `json:"external_url"`
+	LastDeployment Deployment `json:"last_deployment,omitempty"`
```

Comment:
This should be a pointer (`*Deployment`) and the `omitempty` tag is not needed here.